### PR TITLE
Trained model + Model script for ResNet50

### PR DESCRIPTION
**Pull request by dangdd2003**
# Models
- 25 epochs trained model for classify ant and bee
- 30 epochs trained model for classify 535 bird species
# Script
- ant - bee model script
- bird species model script

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
-# DL-Resnet50
-Examples for resnet50 classification
+# DL-Resnet50 - Deep Leerning Project
+
+## Dataset
+kaggle dataset
+- Hymenoptera Dataset (Ant-Bee classification): https://www.kaggle.com/datasets/thedatasith/Hymenoptera
+
+- Birds 535 species Dataset: https://www.kaggle.com/datasets/gpiosenka/100-bird-species
```
Comment (bhhoang) [inline README.md]: *525 species :))

**Review by bhhoang — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dangdd2003) [reply README.md]: thi m sua lai, m sua dc ma :+1:

Comment (bhhoang) [reply README.md]: M sua di

**Review by dangdd2003 — COMMENTED** [review]
fix 535 species -> 525 species

**Review by bhhoang — APPROVED** [review]

**Pull request merged**